angular-bind-html-compile.js

```diff
@@ -1,9 +1,9 @@
 (function (angular) {
     'use strict';
 
-    var module = angular.module('angular-bind-html-compile', []);
+    var mod = angular.module('angular-bind-html-compile', []);
```

Comment:
That's a breaking change for anyone using it via npm though, so you'll have to jump up a version